models/articles.js

```diff
@@ -1,13 +1,33 @@
 const db = require("../db/connection");
 
 exports.obtainArticles = () => {
-  return db.query(`   
+  return db
+    .query(
+      `   
         SELECT articles.author, articles.title, articles.article_id, topic, articles.created_at, articles.votes, COUNT(comment_id) AS comment_count
         FROM articles
         JOIN comments ON comments.article_id=articles.article_id
```

Comment:
This is for a different ticket and may have been missed - but you'll want a LEFT JOIN here - otherwise, articles with no comments will be omitted